http_endpoints.go

```diff
@@ -104,6 +104,11 @@ func BatchEvents(client *devcycle.Client) gin.HandlerFunc {
 
 func GetConfig(client *devcycle.Client) gin.HandlerFunc {
 	return func(c *gin.Context) {
+		if c.Param("sdkKey") == "" || !strings.HasSuffix(c.Param("sdkKey"), ".json") {
```

Comment:
lol, just sneaking this in with the release change?

Reply:
That was why I initially was tweaking it - but thought may as well include it

Reply:
the branch came before the releases